app/settings/environments.js

```diff
@@ -0,0 +1,25 @@
+import {filter} from 'wasmuth'
+
+const environments = {
+  development: ['http://localhost:3333', 'http://localhost:3334'],
+  staging: '[STAGING_URL]',
+  production: '[PRODUCTION_URL]'
+}
+
+export const environment = (() => {
+  const {host, protocol} = window.location
+  const url = `${protocol}//${host}`
+  const current = filter(
+    (k) => {
+      if (typeof environments[k] === 'object') {
+        return filter(v => v === url, environments[k]).length > 0
```

Comment:
Could use some here: https://github.com/staydecent/wasmuth/blob/master/src/some.js

which falls back to https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/some for arrays.

which is similar to http://ramdajs.com/docs/#any